.gitmodules

```diff
@@ -331,3 +331,6 @@
 	url = https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c
 	platforms = esp32
 	recursive = true
+[submodule "third_party/hostap/repo"]
+	path = third_party/hostap/repo
+	url = https://w1.fi/hostap.git
```

Comment:
This seems to give a 404 to me. 
This should probably have a platforms guard to linux.

Reply:
Thank you for your comments @andy31415 , indeed I had not committed the actual submodule. About the repo URL, it is valid for `git clone`, but not suitable for loading with a web browser.

Reply:
Added [this mirror](https://gitlab.com/prpl-foundation/mirrors/hostap) instead. That solved some CI issues with the original `hostap` Git server not supporting shallow clones.

Reply:
https://git.w1.fi/hostap.git is the official upstream repo URL according to https://w1.fi/cvs.html, so we should use that and not a third-party mirror repo. Maybe put a comment like `# home page: https://w1.fi/hostapd/` in there so there's a browsable URL as well.